### PR TITLE
Bubble sort task solved with flask framework

### PR DESCRIPTION
**Pull request by munnyraht**
issue #253 task solved.

### CODE REVIEW

File: bubble sort/bubblesort.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+@app.route('/', methods = ['POST', 'GET'])
+def index():
+    array = [10, 1, 200, -19, 21, 321, 0, 200 ]
+    sorted_array = bubble_sort(array)
+    return json.dumps(sorted_array)
+
+
+if __name__ == '__main__':
+    app.run()
```
Comment (markyjackson-taulia) [inline bubble sort/bubblesort.py:31]: Can you kindly add a line feed here

**Review by markyjackson-taulia — CHANGES_REQUESTED** [review]

**Review by sbtaylor15 — CHANGES_REQUESTED** [review]
Need to comment each line and describe what it is for.

**Comment by munnyraht** [comment]
> Need to comment each line and describe what it is for.

ok

File: bubble sort/bubblesort.py (hunk 1 above)
Comment (munnyraht) [reply bubble sort/bubblesort.py:31]: > Can you kindly add a line feed here

do you mean like a line break?

Comment (markyjackson-taulia) [reply bubble sort/bubblesort.py:31]: yes, at line 31

**Review by markyjackson-taulia — APPROVED** [review]

File: bubble sort/bubblesort.py (hunk 1 above)
Comment (munnyraht) [reply bubble sort/bubblesort.py:29]: ok, done

**Comment by sbtaylor15** [comment]
@munnyraht please add comments as follows to each line.  

`import json  # module to parse json data`

Also, need to add in the pydoc docstrings for the functions.  Here is an example of the [pydoc docstrings format](https://stackabuse.com/python-docstrings/).  Don't worry about the Google docstring format.  We don't use that format.

```
def post_json(url, payload, cookies):
    """
    Post URL as json string.

    Args:
        url (string): url to server
        payload (string): json payload to post
        cookies (string): login cookies

    Returns:
        string: The json string.
    """
    try:
        res = requests.post(url, data=payload, cookies=cookies)
        if (res is None):
            return None
        if (res.status_code != 200):
            return None
        return res.json()
    except requests.exceptions.ConnectionError as conn_error:
        print(str(conn_error))
    return None
```

**Pull request merged**